src/code/views/value-slider-view.tsx

```diff
@@ -217,12 +236,33 @@ export class SVGSliderView extends React.Component<SVGSliderViewProps, SVGSlider
     return <div className="number" style={style}>{this.props.value.toFixed(this.props.displayPrecision)}</div>;
   }
 
+  private isButtonEvent(e) {
+    const {target} = e;
+    const {parentNode} = target;
+    return (target.getAttribute("class") === "slider-button") || (parentNode && (parentNode.getAttribute("class") === "slider-button"));
+  }
+
   private handleNoop = (e) => {
```

Comment:
I know that this commit and PR doesn't modify this method `handleNoop` -- but when I was reading your code it occurred to me that this is a bad name for this method.

The method name should be like `stopPropogation` or something.

Reply:
I agree, or `stopEvent()`